visidata/loaders/json.py

```diff
@@ -187,6 +187,74 @@ def JSON(vd, s:str):
     return json.loads(s)
 
 
+class JsonGuide(GuideSheet):
+    guide_text = '''# Some special features for JSON
+
+## Working with nested data
+
+To expand nested data horizontally (into new columns):
+
+- {help.commands.expand-col}
+- {help.commands.expand-col-depth}
+- {help.commands.expand-cols}
+- {help.commands.expand-cols-depth}
+
+To revert earlier `expand-` operations:
+
+- {help.commands.contract-col}
+- {help.commands.contract-col-depth}
+- {help.commands.contract-cols}
+- {help.commands.contract-cols-depth}
+
+To unfurl (explode) nested data into new rows:
+
+- {help.commands.unfurl-col}
+
+Note that `unfurl-col` creates a new sheet with `_unfurled` appended to the name. There is no command to revert an unfurl, instead close the unfurled sheet.
+
+To "zoom in" on a specific cell:
+
+- {help.commands.open-cell}
+
+## Options to control JSON save behavior
+
+- {help.options.json_indent}
+- {help.options.json_sort_keys}
+- {help.options.json_ensure_ascii}
+
+## Visual cues
+
+VisiData provides visual cues to help identify and preview nested JSON data. Given the following data:
+
+`[`
+`  {{`
+`    "geolocation": {{`
+`        "type":"Point",`
+`        "coordinates": [6.08333,50.775]`
+`    }},`
+`  }},`
+`  {{`
+`    "geolocation": {{`
+`        "type":"Point",`
+`        "coordinates": [2.08,30.77]`
+`    }}`
+`  }}`
+`]`
```

Comment:
This is admittedly weird. Triple backtick and `[:code]...[/]` didn't work for a multi-line code segment though. Maybe there's a good answer, or maybe the better answer is to not try to do this in the first place, and save this sort of thing for posts/tutorials.

Reply:
Trimmed & moved to the top based on suggestions.